generator-tests/aa-million-messages.txt

```diff
@@ -6,6 +6,7 @@ execution-cost: 4801223308
 condition-cost: 2974950000
 removal_amount: 123
 addition_amount: 0
-atoms: 2654
-pairs: 23457
-heap: 29388
+atoms: 2975153
+pairs: 62475440
+heap: 142
```

Comment:
these counters changed quite a bit. Do you know why? I don't think we can allow them to change.
Maybe this test not run before?

Reply:
Yes this test was commented out as a case and was not running so needed a more major update